sdk/storage/azure-storage-blob/azure/storage/blob/_blob_client.py

```diff
@@ -2967,6 +2974,92 @@ def get_page_ranges( # type: ignore
             process_storage_error(error)
         return get_page_ranges_result(ranges)
 
+    @distributed_trace
+    def list_page_ranges(
+            self,
+            *,
+            offset: Optional[int] = None,
+            length: Optional[int] = None,
+            previous_snapshot: Optional[Union[str, Dict[str, Any]]] = None,
+            **kwargs: Any
+        ) -> ItemPaged[PageRange]:
```

Comment:
Thank you for a nice Python3 style type hinted method signature :)

Reply:
Happy to start to be able to use this style :)